cmd/server/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os"
 
 	"github.com/uber/cadence/cmd/server/cadence"
+	//_ "github.com/uber/cadence/common/persistence/sql/sqlplugin/db2"      // needed to load db2 plugin
```

Comment:
Please don't check in commented out code.

Reply:
deleted